### PR TITLE
Feature/ltbi hh tb exposure calc

### PR DESCRIPTION
**Pull request by yongqx**
@halfhorst I made a few changes to the `load_hh_data()` function. Now all household microdata have been extracted and saved as .dta file under `/home/j/Project/simulation_science/latent_tuberculosis_infection/literature/household_structure/microdata/`

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]
Does this mean you have a household data source for each of the five locations now?

**Comment by yongqx** [comment]
> Does this mean you have a household data source for each of the five locations now?

Yes!

**Comment by halfhorst** [comment]
I"m going to merge in my parallel version and then update it with these changes. Thanks!

File: src/vivarium_csu_ltbi/external_data/ltbi_hh_tb_exposure_calculation.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            df['age'] = df['age'].replace('96+', 96)
+        else:
+            df['age'] = df['age'].replace('95+', 95)
+        df = df[df.age != "don't know"]
+    else:
+        df['age'] = df['age'].replace({'Less than 1 year': '0',
+                                       'less than 1 year': '0',
+                                       '1 year': '1',
+                                       '2 years': '2',
+                                       '100+': '100'}).astype(int)
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/external_data/ltbi_hh_tb_exposure_calculation.py:37]: This is the only change, @yongqx  ?

**Comment by halfhorst** [comment]
Addressed in #26

**Pull request closed without merging**